playbook-website/Gemfile.lock

```diff
@@ -150,11 +151,8 @@ GEM
     net-smtp (0.3.3)
       net-protocol
     nio4r (2.5.9)
-    nokogiri (1.14.2-arm64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.14.2-x86_64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.14.2-x86_64-linux)
+    nokogiri (1.14.2)
```

Comment:
did this need to be upgraded because of the ruby version? just curious

Reply:
@markdoeswork 

Great question. It definitely upgraded on it's own.
I just ran ./setup and let bundler upgrade as it saw fit. cc: @markdoeswork 

👍  We'll test it and see if there are any issues. 
In fact, would you mind sanity-checking my PR, by pulling down my branch, and running the commands:
1. `asdf plugin-update ruby`
2. `asdf install ruby 3.3.0`
3. `./setup` OR `bundle && yarn`
4. `yarn start-dev`
5.  Confirm the site loads. :)

Reply:
thats for the deets